data/os/Debian/9.yaml

```diff
@@ -1,4 +1,5 @@
 ---
-snmp::snmpd_options: "-Lsd -Lf /dev/null -u Debian-snmp -g Debian-snmp -I -smux -p /var/run/snmpd.pid"
+snmp::snmpd_options: "/usr/sbin/snmpd -Lsd -Lf /dev/null -u Debian-snmp -g Debian-snmp -I -smux,mteTrigger,mteTriggerConf -f"
```

Comment:
I suppose this value of parameter is wrong because it handle options and here we start with a path of command.
This should raise a failing test.
This highlights we need to add tests in acceptance about content of file given by parameter `snmp::sysconfig`.

Reply:
I don't understand. This value is good for systemd, it needs this line exactly.

Reply:
The name of parameter is `snmp_options` so i am expecting *options* and not path of *binary and options*.

Reply:
So what do you recommends  ? Hard code the binary path and add the variable  ?

Reply:
I think the last commit is what you expect.